ambari-server/src/main/java/org/apache/ambari/server/utils/StageUtils.java

```
@@ -278,6 +242,16 @@ public static String jaxbToString(Object jaxbObj) throws JAXBException,
     return commandParams;
   }
 
+  /**
+   * A helper method for generating keys for the clusterHostInfo section.
+   */
+  public static String getClusterHostInfoKey(String componentName){
+    if (componentName == null){
+      throw new NullPointerException("Component name cannot be null");
```

I'd rather throw an IllegalArgument exception here since method invariant is violated.

---

Thanks, corrected.